k8s/proxy/src/lib.rs

```diff
@@ -16,16 +16,27 @@ pub use proxy::{ConfigBuilder, ForwardingProxy, LokiClient, Scheme};
 /// Get the `kube::Config` from the given kubeconfig file, or the default.
 pub async fn config_from_kubeconfig(
     kube_config_path: Option<PathBuf>,
-) -> anyhow::Result<kube::Config> {
+    context: Option<String>,
+) -> anyhow::Result<kube::Config> {    
+    let options = match context {
+        Some(ctx) =>  KubeConfigOptions {
+            context: Some(ctx),
+            ..Default::default()
+            },
```

Comment:
spacing seems off, did you run `./scripts/rust/linter.sh` ?

Reply:
Opps...Didn't do that.

Will run to fix everything